packages/components/calendar/src/range-date-picker.tsx

```diff
@@ -139,6 +140,7 @@ export const RangeDatePicker = forwardRef<RangeDatePickerProps, "input">(
               <RangeDatePickerField
                 separator={separator}
                 value={value}
+                aria-label="RangeDatePicker"
```

Comment:
Is this `aria-label` necessary?

Reply:
If it is necessary, the wording would be better suited to everyday speech, with the expectation that it will be read out to a screen reader.

Reply:
Thank you for your comment.
I will certainly delete the aria-label as it doesn't seem to fit.